full-stack-tests/core/src/backend/backend.ts

```diff
@@ -100,11 +100,15 @@ async function init() {
 
     // create a basic express web server
     const port = Number(process.env.CERTA_PORT || 3011) + 2000;
-    const server = new IModelJsExpressServer(rpcConfig.protocol);
+    const server = new WebEditServer(rpcConfig.protocol);
```

Comment:
This is fine for now but if we split out the other routes from the WebEditServer that already exist within IModelJsExpressServer we'll need to come up with a new strategy for the tests.